app/config/database.js

```diff
@@ -0,0 +1,46 @@
+const { DefaultAzureCredential } = require('@azure/identity')
+
+function isProd () {
+  return process.env.NODE_ENV === 'production'
+}
+
+const hooks = {
+  beforeConnect: async (cfg) => {
+    if (isProd()) {
+      const credential = new DefaultAzureCredential()
+      const accessToken = await credential.getToken('https://ossrdbms-aad.database.windows.net', { requestOptions: { timeout: 1000 } })
+      cfg.password = accessToken.token
+    }
+  }
+}
+
+const retry = {
+  backoffBase: 500,
+  backoffExponent: 1.1,
+  match: [/SequelizeConnectionError/],
+  max: 10,
+  name: 'connection',
+  timeout: 60000
+}
+
+const dbConfig = {
+  database: process.env.POSTGRES_DB || 'ffc_mpdp_backend',
+  dialect: 'postgres',
+  dialectOptions: {
+    ssl: isProd()
+  },
+  hooks,
```

Comment:
Need to frequently get the token as expires after 24 hours (not configurable).  So using hooks coupled with retry to ensure that connections always have valid credentials.